### PR TITLE
add activation component for new accounts

### LINKED ISSUE #149: Register a new account
I open this ticket as the URL sent by mail (for new account) seems not working
Not sure, but I need to confirm that.

### PR DESCRIPTION
**Pull request by atomfrede**
To which page should we forward the user after successful activation and login? Currently the users stays on the activation page, which feels kind of strange. 

~~Edit: Currently the login modal always stays on the current page (without reloading it). Will open an issue for that, so we can go forward with this.~~ Will be handled most likely in https://github.com/jhipster/jhipster-vuejs/issues/185

~~Edit2: And tests are still missing, will track them too in #175~~ :heavy_check_mark:  

closes #149

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/jhipster-vuejs/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-vuejs/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/files.js [hunk 1]
```diff
@@ -196,7 +196,10 @@ const vueFiles = {
                 'account/reset-password/reset-password.vue',
                 'account/reset-password/reset-password.component.ts',
                 'account/settings/settings.vue',
-                'account/settings/settings.component.ts'
+                'account/settings/settings.component.ts',
+                'account/activate/activate.component.ts',
```
Comment (pascalgrimaud) [inline generators/client/files.js]: can you add npm test for these new files plz ?

Comment (atomfrede) [reply generators/client/files.js]: You mean vue/component tests or just testing that these files are generated?

**Comment by atomfrede** [comment]
I added the missing unit test for the activate component

File: generators/client/files.js (hunk 1 above)
Comment (pascalgrimaud) [reply generators/client/files.js]: I mean npm test with Yeoman, to be sure these files are generated

Comment (atomfrede) [reply generators/client/files.js]: Done in the last commit

**Comment by atomfrede** [comment]
Bounty claimed https://opencollective.com/generator-jhipster/expenses/6346

**Pull request merged**